src/test/java/redis/clients/jedis/tests/collections/SetFromListTest.java

```diff
@@ -0,0 +1,93 @@
+package redis.clients.jedis.tests.collections;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Test;
+
+import redis.clients.collections.SetFromList;
+
+public class SetFromListTest {
+
+  @Test
+  public void setOperations() throws Exception {
+
+    // add
+    SetFromList<String> cut = SetFromList.of(new ArrayList<String>());
+    cut.add("A");
+    cut.add("B");
+    cut.add("A");
+
+    assertEquals(2, cut.size());
+
+    // remove
+    cut.remove("A");
+    assertEquals(1, cut.size());
+
+    cut.remove("C");
+    assertEquals(1, cut.size());
+
+    // contains
+    assertTrue(cut.contains("B"));
+    assertFalse(cut.contains("A"));
+
+    cut.add("C");
+    cut.add("D");
+
+    // containsAll
+    cut.containsAll(cut);
```

Comment:
It seems not affect whole tests. Maybe you missed something to compare, or just missed assertion.